data/mods/Avali/mutations/avali_mutations.json

```
@@ -0,0 +1,267 @@
+[
+  {
+    "type": "mutation",
+    "id": "THRESH_AVALI",
+    "name": "Avali",
+    "points": 1,
+    "description": "This isn't your world, but you might be able to call it home after some cleaning up.",
+    "valid": false,
+    "purifiable": false,
+    "threshold": true
+  },
+  {
+    "type": "mutation",
+    "id": "AVI_EYES",
+    "name": { "str": "Avali Eyes" },
+    "points": 0,
+    "visibility": 3,
+    "ugliness": 1,
+    "description": "Your eyes have mutated; now you have big, brightly colored eyes. It's visually striking, but doesn't alter your vision.",
+    "social_modifiers": { "lie": 2, "persuade": 2 },
+    "types": [ "EYES" ],
+    "category": [ "AVALI" ],
+    "threshreq": [ "THRESH_AVALI" ]
+  },
+  {
+    "type": "mutation",
+    "id": "TAIL_STUB",
+    "copy-from": "TAIL_STUB",
+    "extend": { "category": [ "AVALI" ], "changes_to": [ "AVI_TAIL" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "AVI_TAIL",
+    "name": { "str": "Avali Tail" },
+    "points": 1,
+    "visibility": 6,
+    "ugliness": 2,
+    "description": "You have a long, thin tail, tipped with a large feathery spade.  It improves your balance, making your ability to dodge higher, but prevents wearing non-fabric pants.",
+    "types": [ "TAIL" ],
+    "prereqs": [ "TAIL_STUB" ],
+    "category": [ "AVALI" ],
+    "restricts_gear": [ "leg_l", "leg_r" ],
+    "allowed_items": [ "ALLOWS_TAIL" ],
+    "allow_soft_gear": true,
+    "dodge_modifier": 2,
+    "threshreq": [ "THRESH_AVALI" ]
+  },
+  {
+    "type": "mutation",
+    "id": "AVI_EARS1",
+    "name": { "str": "Forked Ears" },
+    "points": 0,
+    "visibility": 4,
+    "ugliness": 2,
+    "description": "Your ears have forked at their tips. Aside from looking rather strange, it doesn't seem to have any effect.",
+    "types": [ "EARS" ],
+    "changes_to": [ "AVI_EARS2" ],
+    "category": [ "AVALI" ]
+  },
+  {
+    "type": "mutation",
+    "id": "AVI_EARS2",
+    "name": { "str": "Avali Ears" },
+    "points": 2,
+    "visibility": 6,
+    "ugliness": 2,
+    "description": "You have four ears instead of two, and they wiggle and twist towards sounds.  Your ability to detect distant noises is quite impressive.",
+    "types": [ "EARS" ],
+    "prereqs": [ "AVI_EARS1" ],
+    "category": [ "AVALI" ],
+    "hearing_modifier": 2.5,
+    "threshreq": [ "THRESH_AVALI" ]
+  },
+  {
+    "type": "mutation",
+    "id": "AVI_BLOOD",
+    "name": { "str": "Avali Metabolism" },
+    "points": 0,
+    "mixed_effect": true,
+    "description": "Your muscle response is dependent on ambient temperatures.  You lose 1% of your speed for every 5 (2.8 C) degrees above 65 F (18.3 C). Your metabolism is more effective, however.",
```

```suggestion
    "description": "Your muscle response is dependent on ambient temperatures.  You lose 1% of your speed for every 5 (2.8) degrees above 65 F (18.3 C). Your metabolism is more effective, however.",
```

Sorry that was wrong.

---

Why wrong? It seems better with the C in there to me.

---

Either
`every 5 F (2.8 C) degrees above 65 F (18.3 C)`
or
`every 5 (2.8) degrees above 65 F (18.3 C) respectively`

---

I like the former. The 'respectively' might make people have to backtrack to match the pairs. I'll leave it with the former styling.